headless-services/spring-boot-language-server/src/main/java/org/springframework/ide/vscode/boot/java/livehover/v2/SpringProcessConnectorService.java

```diff
@@ -292,4 +297,130 @@ private void scheduleRefresh(IndefiniteProgressTask progressTask, SpringProcessP
 	private IndefiniteProgressTask getProgressTask(String prefixId, String title, String message) {
 		return this.progressService.createIndefiniteProgressTask(prefixId + progressIdKey++, title, message);
 	}
+	
+	public void getLoggers(SpringProcessParams springProcessParams) {
+		log.info("get loggers data: " + springProcessParams.getProcessKey());
+
+		SpringProcessConnector connector = this.connectors.get(springProcessParams.getProcessKey());
+		if (connector != null) {
+			final IndefiniteProgressTask progressTask = getProgressTask(
+					"spring-process-connector-service-get-loggers-data" + springProcessParams.getProcessKey(), "Loggers", null);
+			System.out.println(progressTask);
```

Comment:
Watch out for these `System.out.println` statements in the code :-) They are okay when you connect to LS via socket but once you build a VSCode extension and launch it without connecting to LS the comm goes over System in/out and that print statement would just corrupt the JSON RPC comm and shutdown the LS for unknown reason ;-)
So, yes, these statement would need to go or replaced with log statement :-)
I'm guilty for leaving `System.out.println` multiple times ;-)